backend/src/db/migrations/20250405030454-email-notification-settings.ts

```diff
@@ -0,0 +1,68 @@
+import { getDriver } from '../neo4j'
```

Comment:
I think this makes the error in the pipeline import following should not make the error:

```suggestion
import { getDriver } from '@db/neo4j'
```